src/elements/subsquid/Subsquid.wc.svelte

```diff
@@ -0,0 +1,176 @@
+  import DeployBtn from "../../components/DeployBtn.svelte";
+  import Alert from "../../components/Alert.svelte";
+  import SelectNodeId from "../../components/SelectNodeId.svelte";
+  import Modal from "../../components/DeploymentModal.svelte";
+
+  // utils
+  import hasEnoughBalance from "../../utils/hasEnoughBalance";
+  import validateName, { isInvalid ,validateEndpoint} from "../../utils/validateName"; // prettier-ignore
+  import { noActiveProfile } from "../../utils/message";
+
+  let data = new Subsquid();
+  let profile: IProfile;
+
+  let loading = false;
+  let success = false;
+  let failed = false;
+
+  let status: "valid" | "invalid";
+
+  const deploymentStore = window.configs?.deploymentStore;
+  const currentDeployment = window.configs?.currentDeploymentStore;
+
+  // Tabs
+  const tabs: ITab[] = [{ label: "Base", value: "base" }];
+  let active = "base";
+
+  // Fields
+  // prettier-ignore
+  const fields: IFormField[] = [
+    { label: "Name", symbol: "name", placeholder: "Subsquid Instance Name", type: "text", validator: validateName, invalid: false },
+    { label: "Subsquid Endpoint", symbol: "endPoint", placeholder: "Subsquid Endpoint", type: "text",validator: validateEndpoint,  invalid: false },
+    { label: "Planetary Network", symbol: "planetary", placeholder: "Enable planetary network", type: 'checkbox' },
+    { label: "Public IP", symbol: "publicIp", placeholder: "Enable Public Ip", type: 'checkbox' },
+  ];
+
+  $: disabled = ((loading || !data.valid) && !(success || failed)) || !profile || status !== "valid" || isInvalid([...fields]); // prettier-ignore
+
+  let message: string;
+  let modalData: Object;
+
+  async function deploySubsquidHandler() {
+    loading = true;
+    success = false;
+    failed = false;
+    message = undefined;
+
+    if (!hasEnoughBalance()) {
+      failed = true;
+      loading = false;
+      message =
+        "No enough balance to execute! Transaction requires 2 TFT at least in your wallet.";
+      return;
+    }
+
+    deploySubsquid(data, profile)
+      .then((data) => {
+        modalData = data.deploymentInfo;
+        deploymentStore.set(0);
+        success = true;
+      })
+      .catch((err: string) => {
+        failed = true;
+        message = err;
+      })
+      .finally(() => {
+        loading = false;
+      });
+  }
+
+  $: logs = $currentDeployment;
+</script>
+
+<SelectProfile
+  on:profile={({ detail }) => {
+    profile = detail;
+  }}
+/>
+
+<div style="padding: 15px;">
+  <form class="box" on:submit|preventDefault={deploySubsquidHandler}>
+    <h4 class="is-size-4 mb-4">Deploy a Subsquid Instance</h4>
+    <p>
+      Subsquid indexer is a piece of software that reads all the blocks from a
+      Substrate based blockchain, decodes and stores them for processing in a
+      later stage.
+      <a
+        target="_blank"
+        href="https://library.threefold.me/info/manual/#/manual__weblets_presearch"
```

Comment:
another url maybe?

Reply:
As there's no current documentation for subsquid in the threefold manual, Maybe we should remove the reference link for now and added later on when the documentation is available?

Reply:
please remove the link for now and create 2 issues
- one on the manual https://github.com/threefoldfoundation/info_manual3/issues to add docs for the solution
- and the other here in weblets to add the link back once the docs are added

Reply:
please remove the link for now and create 2 issues
- one on the manual https://github.com/threefoldfoundation/info_manual3/issues to add docs for the solution
- and the other here in weblets to add the link back once the docs are added

Reply:
Done